website/content/api-docs/secret/aws.mdx

```diff
@@ -329,6 +335,22 @@ Using groups:
 }
 ```
 
+Using tags:
```

Comment:
nit: this could be improved with tabs: https://github.com/hashicorp/vault/tree/master/website#tabs

Reply:
This should have been addressed